### PR TITLE
Rewrite for 2.0

### PR DESCRIPTION
**Pull request by ctjhoa**

### CODE REVIEW

File: addon/components/aria-tabs.js [hunk 1]
```diff
@@ -1,6 +1,370 @@
...
+    // No tabs are disabled, return index
+    return index;
+  },
+
+  getFirstTab() {
+    let tabsNodes = get(this, 'tabNodes');
+    const count = tabsNodes.length;
+
+    // Look for non disabled tab from the first tab
+    for (let i = 0; i < count; i++) {
```
Comment (thiberaw) [inline addon/components/aria-tabs.js]: pourquoi ne pas faire un filter sur l'array pour n'avoir que les non disabled et `return array[0] || null;` ?

File: addon/components/aria-tabs.js [hunk 2]
```diff
@@ -1,6 +1,370 @@
...
+    }
+
+    return null;
+  },
+
+  getLastTab() {
+    let tabsNodes = get(this, 'tabNodes');
+    let i = tabsNodes.length;
+
+    // Look for non disabled tab from the last tab
```
Comment (thiberaw) [inline addon/components/aria-tabs.js]: idem mais avec `return array[array.length - 1] || null;`

File: addon/components/aria-tab.js [hunk 3]
```diff
@@ -1,6 +1,222 @@
...
+  panelId: computed('nodeIndex', 'panelNodes.[]', function() {
+    let {
+      nodeIndex,
+      panelNodes
+    } = getProperties(this, [
+      'nodeIndex',
+      'panelNodes'
+    ]);
+    let panel = panelNodes[nodeIndex];
+    return panel? panel.id : null;
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply addon/components/aria-tab.js]: This has been ported as is from react-tabs
https://github.com/reactjs/react-tabs/blob/master/src/components/UncontrolledTabs.js

File: addon/components/aria-tabs.js (hunk 1 above)
Comment (ctjhoa) [reply addon/components/aria-tabs.js]: This has been ported as is from react-tabs
https://github.com/reactjs/react-tabs/blob/master/src/components/UncontrolledTabs.js

File: addon/components/aria-tabs.js (hunk 2 above)
Comment (ctjhoa) [reply addon/components/aria-tabs.js]: This has been ported as is from react-tabs
https://github.com/reactjs/react-tabs/blob/master/src/components/UncontrolledTabs.js

**Pull request merged**